### PR TITLE
WIP: update README

### PR DESCRIPTION
**Pull request by cportele**
This is the start of the edits to update the README landing page of the repository as discussed in #76 and #93. 

This is work in progress.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,117 @@
...
 
-This is a new OGC working group and the work has started on June 1st, 2021. A first draft of the candidate standard, version 0.1.0, is available.
+The current version is 0.2.2. 
+
+This version will be the basis for the OGC Public Comment phase for 90 days, which is the final call for comments before the candidate standard will go through the process for approval as an OGC Standard after resolving the comments received during the OGC Public Comment period.
 
 The editor's draft of the specification can be found at [docs.ogc.org/DRAFTS/21-045.html](https://docs.ogc.org/DRAFTS/21-045.html). A [PDF version](https://docs.ogc.org/DRAFTS/21-045.pdf) is available, too.
 
-Next steps:
+The Open Geospatial Consortium (OGC) invites organisations and developers that have a need for the extensions specified by this specification to implement and test the extensions. Please submit feedback in the [in this repository](https://github.com/opengeospatial/ogc-feat-geo-json/issues). Are these extensions useful for your use cases? Are they simple enough to implement?
```
Comment (lvdbrink) [inline README.md:15]: remove "in the" just before "[in this repository]

File: README.md [hunk 2]
```diff
@@ -1,47 +1,117 @@
...
+
+The JSON-FG Standard specifies the following extensions to the GeoJSON format:
+
+* The ability to use Coordinate Reference Systems (CRSs) other than WGS84 (OGC:CRS84);
+* Allows the use of non-Euclidean metrics, in particular ellipsoidal metrics;
+* Support for solids and prisms as geometry types;
+* The ability to encode temporal characteristics of a feature; and
+* The ability to declare the type and the schema of a feature.
+
+Geographic features, their properties, and their spatial extents that can be represented as GeoJSON objects are encoded as GeoJSON. Additional information not specified in the GeoJSON RFC is mainly encoded in additional members of the GeoJSON objects. The additional members use keys that do not conflict with existing GeoJSON keys. This was done so that existing and future GeoJSON clients can continue to successfully parse and understand GeoJSON encoded content. JSON-FG enabled clients will also be able to parse and understand the additional members.
```
Comment (lvdbrink) [inline README.md:37]: "the GeoJSON RFC" > standards experts / spec writers will know what this means but others maybe not. Suggestions: replace with "the GeoJSON standard" or similar wording.

**Review by lvdbrink — CHANGES_REQUESTED** [review]
I left two editorial remarks. Otherwise, it looks good!

File: README.md (hunk 1 above)
Comment (cportele) [reply README.md]: removed "in the"

File: README.md (hunk 2 above)
Comment (cportele) [reply README.md]: changed as proposed

**Review by lvdbrink — APPROVED** [review]

**Comment by lvdbrink** [comment]
I found one more typo. Since I can't edit the branch myself, I'll leave it as a comment here:

In the implementations table, in the description of ShapeChange:
> accoring to ISO 19109.

should be 'according'.

**Comment by cportele** [comment]
Thanks @lvdbrink - I fixed it.

**Comment by lvdbrink** [comment]
Another implementation can be added: 

| [Imvertor]( https://armatiek.nl/imvertor.html) | Stable | Imvertor processes information models modelled in UML according to (Dutch) [metamodel for information modeling](https://docs.geostandaarden.nl/mim/mim/). The [JSON Schema target](https://shapechange.github.io/ShapeChange/3.0.0/targets/JSON_Schema.html) supports generating schemas for validating JSON-FG or GeoJSON instances. | X | X | X | - | X | - |

**Pull request merged**